src/test/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/integration/IntegrationTestBase.kt

```diff
@@ -46,4 +48,14 @@ abstract class IntegrationTestBase {
     roles: List<String> = listOf(),
     scopes: List<String> = listOf(),
   ): (HttpHeaders) -> Unit = jwtAuthHelper.setAuthorisation(user, roles, scopes)
+
```

Comment:
Is this because we have some tests that are not SQS test but require localstack?

Reply:
Yes sort of, It's basically because the services have the retry service as a dependency, e.g. activitiesService requires activitiesRetryQueueService